docs/find_and_replace_action_json_patch.md

```diff
@@ -0,0 +1,595 @@
+- When you need to use data that is already inside the JSON Document and move it around. Sometimes `fixing` a value, in the sense of putting a static replacement, is not what you need. Other `Actions` [documented](find_and_replace.md) will allow you to replace one value with another fixed one. But JSON Patch will allow you to use existing data inside your target JSON and move it/copy it.
+- When you operate on multiple JSON Keys and can target / match complete values. You can not replace a single word inside a value via a JSON Patch. But you can apply multiple precise operations in a single pass.
+- When you need to change data types. e.g. convert a single value into an array.
+- When you need safety. The fact that a single failed  “test“  will cancel a whole JSON Patch allows you to operate with safety and ensure the final destination will always be a JSON
+
+Now that we know what it is and when we should do it, we can make a small exercise. The goal:
+
+- For all ADO's with json key with value `photograph` that are member of collection with Node ID `16`
+    - convert the `description` key from a single value into an array.
+    - add an extra LOD entry
+    - Copy a Value into a new Key
+
+## Step 1:  Select the ADOs to be Modified
+
+As with every other VBO action described in our documentation, start by selecting the ADOs you want to modify using the exposed Search Field(s) and/or Facets present in the Search and Replace `View` found at `/search-and-replace` . 
+
+Once you have filtered down the list to manageable size, containing at least the ADOs you plan on modifying (but for JSON Patch operation could be more too because you can “test“ and match ), press either on `Select / deselect all results in this view` to pass all  the result (this includes all pages, not only the currently visible one) or go selectively one by one by checking on the `toggle` found beside each ADO's title. You will see how the number in `Selected 0 item in this view` increases. Now press `Apply to select Items`.  We will use for this example `Laddie the dog running in the garden, Bronx, N.Y., undated [c. 1910-1918?]`, an ADO we provide in our DEMO sets. 
+
+Redundant to say, Batch Actions are intended to be used when a modification needs to be applied to more than one item, implying there is a pattern. For single ADOs you can always do this faster directly via the **EDIT** tab.
+
+??? note "Keep your JSON Patches (and friends) around"
+
+	Since JSON Patching involves writing a, sometimes complex, JSON Document, please keep around an Application (or Text File) where you can copy/paste and save your JSON Patches for resuse or future references. Archipelago will not store nor remember between runs the JSON Patch document you submitted. It is also very useful to copy and have at hand the `RAW JSON` of one of the ADOs you plan to modify as a reference/aid while building the given JSON Patch document.
```

Comment:
```suggestion
    Since JSON Patching involves writing a, sometimes complex, JSON Document, please keep around an Application (or Text File) where you can copy/paste and save your JSON Patches for resuse or future references. Archipelago will not store nor remember between runs the JSON Patch document you submitted. It is also very useful to copy and have at hand the `RAW JSON` of one of the ADOs you plan to modify as a reference/aid while building the given JSON Patch document.
```